misk/src/main/kotlin/misk/hibernate/Transacter.kt

```diff
@@ -1,5 +1,23 @@
 package misk.hibernate
 
+/**
+ * Provides explicit block-based transaction demarcation.
+ */
 interface Transacter {
+  /**
+   * @returns [true] if the calling thread is currently within a transaction block.
```

Comment:
nit: `@return`, or even better `Returns`

`@return` is a useful tag when there’s a lot of primary documentation and you want to go in detail on how to interpret the return value. But here the entire point of the method is to return something, so there’s no reason to hide the documentation under the `@return` tag.

```
/** 
 * Returns true if the calling thread is currently within a transaction block.  
 */
```

Reply:
Also, we only wanna use `code blocks` and `[braces]` when they disambiguate. For words like true, null, etc., we want to use plain English as much as possible. (And if the reader wants the precision of code, they have the source right there)